op-service/sources/l2_client.go

```diff
@@ -26,6 +26,8 @@ type L2ClientConfig struct {
 	L1ConfigsCacheSize   int
 
 	RollupCfg *rollup.Config
+
+	MessagePasserRootFromState bool
```

Comment:
```suggestion
	MessagePasserRootFromState bool

	RollupCfg *rollup.Config
```
nit, I'd group it with the other behavior-changing config parameters